builder/tests/builder_test.go

```diff
@@ -65,4 +79,6 @@ func TestBuilder(t *testing.T) {
 	dockercli.DeisServiceTest(t, name, port, "tcp")
 	etcdutils.VerifyEtcdValue(t, "/deis/builder/host", host, etcdPort)
 	etcdutils.VerifyEtcdValue(t, "/deis/builder/port", port, etcdPort)
+
+	defer os.Remove(tmpfile.Name())
```

Comment:
This should be moved up to line 45. That way, the next person reading this won't be wondering where the call to `defer` is hiding.